src/lib/bootloader/generic_widgets.rb

```diff
@@ -43,6 +43,7 @@ def localized_names(name)
       names = {
         "grub2"        => _("GRUB2"),
         "grub2-efi"    => _("GRUB2 for EFI"),
+        "grub2-bls"    => _("GRUB2 with BLS"),
```

Comment:
I think it would deserve comment for translators as BLS akak BootLoader Specification is not much widely known term